Nos/Assets/Colors.xcassets/action-banner-button-gradient-leading.colorset/Contents.json

```diff
@@ -5,9 +5,9 @@
         "color-space" : "srgb",
         "components" : {
           "alpha" : "1.000",
-          "blue" : "0xF7",
-          "green" : "0xF8",
-          "red" : "0xFF"
+          "blue" : "0.969",
+          "green" : "0.973",
+          "red" : "1.000"
```

Comment:
I think we want to keep the hex values since our designs are always specified in hex. Same applies to all the others.